lib/home_page.dart

```diff
@@ -14,7 +20,34 @@ class _HomePageState extends State<HomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(widget.title),
+        title: Text(GlobalLocalizations.of(context).chatApp),
+      ),
+      body: Column(
+        children: <Widget>[
+          MaterialButton(
+            child: Text(
+              'Polski',
+              style: TextStyle(color: Theme.of(context).accentColor),
+            ),
+            onPressed: () {
+              Provider.of<LocaleModel>(context, listen: false)
+                  .changeLocale(Locale('pl'));
+              Provider.of<ThemeModel>(context, listen: false).changeTheme(
+                ChatAppThemes.defaultTheme.copyWith(accentColor: Colors.blue),
+              );
+            },
+          ),
+          MaterialButton(
```

Comment:
Może zrobić nową klasę (np. language button, gdzie podajesz kod/id języka i zmienia on język na podany?

Reply:
Bardziej myślałem o jakiejś rozwijanej liście wspieranych języków do wyboru w oddzielnym route, pytanie gdzie chcielibyśmy umieścić taki button typu 'wybierz język' który by przenosił do tej listy

Reply:
lista byłaby spoko, myślę że ścieżka to po prostu settings -> language